geoips/schema/bases/unvalidated.yaml

```diff
@@ -0,0 +1,10 @@
+$id: bases.unvalidated
```

Comment:
What is the difference between bases.unvalidated and bases.product_defaults.unvalidated?